index.html

```diff
@@ -218,6 +217,25 @@ <h1>hackSocIndore</h1>
     </div>
     <!-- End Culture -->
 
+    <!--========== Contact Us ==========-->
+
+    <div style="background-color: lightgray; padding-top: 50px; padding-bottom: 60px;">
+
+        <div class="section section--xTall">
+            <div class="wrapper wrapper--thin">
```

Comment:
do we have `wrapper` and `wrapper--thin` classes?

Reply:
I don't think they are needed , should remove them